.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -1,26 +1,36 @@
-name: Bug Report
-description: Report a bug/issue that you found in the SVGs, CSS files, etc.
-title: '[BUG] <bug name>'
-labels: [bug]
-body:
-  - type: textarea
-    id: description
-    attributes:
-      label: Bug description
-      description: Short description of the bug that you found. Provide images/code if applicable.
-    validations:
-      required: true
-  - type: textarea
-    id: howtoreplicate
-    attributes:
-      label: Possible fixes or solutions
-      description: List any possible fixes/suggestions that you have in mind that could solve this issue.
-    validations:
-      required: true
-  - type: textarea
-    id: extrainformation
-    attributes:
-      label: Additional information
-      description: Is there anything else we should know about this bug?
-    validations:
-      required: false
+name: Bug Report
+description: Report a bug/issue that you found in the SVGs, CSS files, etc.
+title: '[BUG] <bug name>'
+labels: [bug]
+body:
+  - type: markdown
+    attributes:
+      value: "## Before you continue, please search our open/closed issues to see if a similar issue has been addressed."
+
+  - type: checkboxes
+    attributes:
+      label: I have searched through the issues and didn't find my problem.
+      options:
+        - label: Confirm

```

Comment:
```suggestion
        - label: Confirm
    validations:
      required: true
```

Reply:
Hi Pan, I'm not too sure what's the change here. Everything looks the same to me. Was it an end line or something?

Reply:
No 😅 This will make the checkbox obligatory to mark (see `validations`, `required: true`).

Reply:
Ah I see, I was confused by the color coding. I tried to do that before too but GitHub doesn't support this [attribute](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema#checkboxes). I can however make that specific box mandatory.

Reply:
> Ah I see, I was confused by the color coding. I tried to do that before too but GitHub doesn't support this [attribute](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema#checkboxes). I can however make that specific box mandatory.

Yes, that'd be great, thanks! 👍